wxt_cidrbot/cmd_list.py

```diff
@@ -115,9 +122,19 @@ def message_handler(self, request, event_type, room_id, pt_id):
                 return self.help_menu(help_word)
         if self.similar(sim_text, "help") > 0.8:
             return self.help_menu("all")
+        if "add repo" in text or "remove repo" in text:
+            if event_type == "Message":
+                for user in self.webex_mod_status:
+                    if user.isModerator:
+                        request = text_split[1] + " " + text_split[2]
+                        name = text_split[3]
+                        return self.dynamo.dynamo_db(request, name, None, None)
```

Comment:
General good practice thing: usually you don't want to trust input from users (even moderators).  So for example, you should at least verify that `name` matches a regex for a git repo.  We can chat about that today if you want.

Reply:
Will be resolved in a follow-on PR